### PR TITLE
Support importing Keystore from device storage

### PR DESCRIPTION
**Pull request by tonybka**
Support importing Keystore from device storage:
- Ask reading external storage permission at runtime
- Open file selector
- Read file and update edit text content

### CODE REVIEW

File: app/src/main/java/com/wallet/crypto/trustapp/ui/ImportKeystoreFragment.java [hunk 1]
```diff
@@ -39,24 +56,127 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
...
+        Log.d(TAG, "File Uri: " + uri.toString());
+        // Get the path
+        if (uri != null) {
+            TaskGetSelectedFileContent taskGetSelectedFileContent = new TaskGetSelectedFileContent(uri);
+            taskGetSelectedFileContent.execute();
+        }
+
+    }
+
+    private class TaskGetSelectedFileContent extends AsyncTask<Void, Void, String> {
```
Comment (madcake) [inline app/src/main/java/com/wallet/crypto/trustapp/ui/ImportKeystoreFragment.java]: We use other approaches for background tasks.

**Comment by vikmeup** [comment]
@tonybka I would suggest before working on a specific task in the project, make sure you follow all the styling guidelines, design pattern and fully understand the issue you are tackling.

File: app/src/main/java/com/wallet/crypto/trustapp/ui/ImportKeystoreFragment.java (hunk 1 above)
Comment (tonybka) [reply app/src/main/java/com/wallet/crypto/trustapp/ui/ImportKeystoreFragment.java]: Thanks @madcake for your instruction.

**Comment by tonybka** [comment]
OK. Noted on that, i need to spend more time for reading our source code before continue doing further. Thanks @vikmeup.

**Pull request closed without merging**